WiseUnpacker/SabreTools.Serialization.Deserializers/WiseSectionHeader.cs

```diff
@@ -87,41 +92,210 @@ public class WiseSectionHeader : BaseBinaryDeserializer<SectionHeader>
             //Seek back to the beginning of the section
             data.Seek(initialOffset, 0);
 
-            header.UnknownValue0 = data.ReadUInt32LittleEndian();
+            header.UnknownDataSize = data.ReadUInt32LittleEndian();
             header.SecondExecutableFileEntryLength = data.ReadUInt32LittleEndian();
             header.UnknownValue2 = data.ReadUInt32LittleEndian();
             header.UnknownValue3 = data.ReadUInt32LittleEndian();
             header.UnknownValue4 = data.ReadUInt32LittleEndian();
             header.FirstExecutableFileEntryLength = data.ReadUInt32LittleEndian();
             header.MsiFileEntryLength = data.ReadUInt32LittleEndian();
-            if (headerLength == 6)
-                return header;
+            // Read common values
```

Comment:
I think this comment got misplaced after the block instead of before.

Reply:
Should be fixed.